host_classifier.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+if [ "$#" -lt 1 ]; then
+echo "please provide at least 1 file"
+fi
+
+var1="0"
+var2="0"
+var3="0"
+var4="0"
+var5="0"
+var6="0"
+
+for each in $*
```

Comment:
above you use the syntax `if []; then` and here you're doing ```
for each in $*
do
```
instead you should be consistent with your style